.circleci/config.yml

```diff
@@ -136,8 +150,9 @@ workflows:
       - test-node-6
       - test-node-8
       - test-node-10
+      - test-node-12
       - test-jest-circus
-      - test-node-11 # current
+      - test-node-11
```

Comment:
Node 11 EOL ist 2019-06-01, can replace it with 12 in case this isn't merged until then, otherwise can remove it in another PR

Reply:
@jeysal I created https://github.com/facebook/jest/pull/8686 to update `jest-leak-detector` package, how I run to test my PR on `test-node-12`?

Reply:
you can cherry-pick the commit from this PR to run on node 12 on yours

Reply:
I'll do what @jeysal suggested though and ditch 11 here